### PR TITLE
Support duplicate property names as array #100

### PR DESCRIPTION
**Pull request by Leask**
- Support duplicate property names as array #100
- fixed all CI error
- tweak some scripts

Related to this PR: https://github.com/npm/ini/pull/100, but fixed all mirror issues.

### CODE REVIEW

**Review by wujunchuan — APPROVED** [review]
LGTM

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (chapterjason) [inline .vscode/settings.json:1]: You shouldn't add IDE specific settings to a repository.

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@
 /coverage
 /node_modules
 /npm-debug.log
+/.history
```
Comment (chapterjason) [inline .gitignore:5]: What kind of file/directory is that?

**Review by chapterjason — APPROVED** [review]
Just two notes.

File: .vscode/settings.json (hunk 1 above)
Comment (Leask) [reply .vscode/settings.json]: Removed.

**Review by chapterjason — APPROVED** [review]
👍🏼

**Comment by Leask** [comment]
Thanks!

**Comment by Leask** [comment]
It's been a long time. 😟

**Review by gregdolley — APPROVED** [review]

**Comment by wraithgar** [comment]
Rewritten in https://github.com/npm/ini/pull/137

**Pull request closed without merging**